README.md

```diff
@@ -3,6 +3,8 @@
 [![Nightly Ubuntu SGX1](https://github.com/alibaba/inclavare-containers/workflows/Nightly%20Ubuntu%20SGX1/badge.svg?branch=master)](https://github.com/alibaba/inclavare-containers/actions?query=workflow%3A%22Nightly+Ubuntu+SGX1%22)
 [![Nightly CentOS SGX1](https://github.com/alibaba/inclavare-containers/workflows/Nightly%20CentOS%20SGX1/badge.svg?branch=master)](https://github.com/alibaba/inclavare-containers/actions?query=workflow%3A%22Nightly+CentOS+SGX1%22)
 
+Inclavare, pronounced [ˈinklɑveə], is the Latin etymology of the word `enclave`, which means to isolate a trusted execution environment for user's workloads in a complex environment where the infrastructure is uncontrollable, in order to meet the user's needs for the security isolation of data in use.
```

Comment:
Change `isolate a trusted execution environment for user's workloads in a complex environment where the infrastructure is uncontrollable` to `isolate the user's sensitive workload from the untrusted and uncontrollable  infrastructure in order to meet the protection requirement for the data in use.`